### PR TITLE
[RFR][1LP] Add _param_name for parametrization

### PR DESCRIPTION
**Pull request by psav**
_param_name inclusion for parametrization

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 367374dc34fcb19135f77942cc20088814cc0d51:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8608937e9de79a651f68d7eecff90c85c2f073cc:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ba834ed5e364f51ecfab84b7e3db1440367b5bda:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9f78de9d184788241496a8935d6566660ffbf74e:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 2eb31301847f4891caa96bbac42469bbdc5ecbe9:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

`fixtures/xunit_tools.py`:
- :bangbang: Line 56:101: **E501** *line too long (128 > 100 characters)*
- :red_circle: Line 94:17: **F841** *local variable 'param_names' is assigned to but never used*
- :bangbang: Line 98:21: **E123** *closing bracket does not match indentation of opening bracket's line*
- :bangbang: Line 101:23: **E702** *multiple statements on one line (semicolon)*
- :grey_exclamation: Line 102:13: **T001** *print statement found.*
- :grey_exclamation: Line 105:17: **T001** *print statement found.*
- :grey_exclamation: Line 107:13: **T001** *print statement found.*
- :grey_exclamation: Line 157:17: **T001** *print statement found.*

Hmmm
:frowning: You might like to check the lint of your code :point_up: 


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 870c3978f23c8ea4f709398e4e6a7906f9e761df:

`cfme/cloud/provider/__init__.py`:
- :red_circle: Line 29:1: **F401** *'utils.ParamClassName' imported but unused*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by mfalesni** [comment]
@psav I think I will need you to shed a bit of light at the changes in this PR, notable the _param_name and then the supposed interaction with pytest, I can't see the connection :)

File: cfme/tests/control/test_basic.py [hunk 1]
```diff
@@ -78,7 +79,11 @@
     conditions.ProviderCondition
 ]
 
-POLICIES_AND_CONDITIONS = zip(CONTROL_POLICIES, CONDITIONS)
+PolicyAndCondition = namedtuple('PolicyAndCondition', ['name', 'policy', 'condition'])
+POLICIES_AND_CONDITIONS = [
+    PolicyAndCondition(name=obj[0].__name__, policy=obj[0], condition=obj[1])
+    for obj in zip(CONTROL_POLICIES, CONDITIONS)
+]
```
Comment (mshriver) [inline cfme/tests/control/test_basic.py]: :+1:

**Comment by psav** [comment]
@mfalesni Yes, right now the xunit tool is not uploaded yet, I will add it shortly, suffice to say that the ```_param_name``` is the name that we want that parameter object to have when it is submitted as part of an iteration to test management systems.

**Comment by mfalesni** [comment]
@psav "parameter object" --> value of the parameter from parametrization?

**Comment by psav** [comment]
@mfalesni yes

**Pull request merged**